### DISCUSSION TITLE
Option to Disable all CSS 💄  in Documentation 📝  Examples

### DISCUSSION THREAD

**Discussion by manavm1990** [radix-ui/primitives#2521] [Ideas]
Currently, there are great options for each example to show the CSS files or Tailwind. What would really drive the point home about what these primitive components really are is an option for 'Raw Primitive.' This would remove all the styling and show what the stuff looks like without any CSS 💄 .

**Comment by joaom00** [comment]
Since Primitives have no style, 'Raw Primitive' will look like native elements

**Reply by manavm1990** [reply]
Yeah. Just add it to to the dropdown to drive home the point. It's a low priority.

**Comment by benoitgrelard** [comment]
To be perfectly honest, in most cases, it would look like absolute horse-shit so I don't think it's worth it.
